docker-compose.yml

```diff
@@ -106,6 +106,8 @@ services:
           - portal
       # app service needs to be on default network too so it can connect to mysql and solr
       default:
+    depends_on:
```

Comment:
Seems like this change to docker-compose is parallel to this fix.

I don't think it matters, just pointing it out.

Reply:
Yeah I added it, because I started a shell for the tests with:
`docker-compose run app /bin/bash`
And that doesn't work unless mysql is running. With the `depends_on` it starts up automatically.